string prefix_from
uint32 key_from
string prefix_to
uint32 key_to
# attitude quaternion coordinates in radians:
float64 qw
float64 qx
float64 qy
float64 qz
# whether the user has confirmed this request
bool confirmed
---
bool success
bool confirm
